analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/LogAnalyzerTest.cs

```diff
@@ -84,10 +98,17 @@ public void LogAutogenerated_VB()
                 VerifyGenerated);
         }
 
-        private static void VerifyCompilationMessages(IEnumerable<LogInfo> messages)
+        private static void VerifyCompilationMessagesNonConcurrentRuleExecution(IEnumerable<LogInfo> messages) =>
+            VerifyCompilationMessagesBase(messages, false);
+
+        private static void VerifyCompilationMessagesConcurrentRuleExecution(IEnumerable<LogInfo> messages) =>
+            VerifyCompilationMessagesBase(messages, true);
+
+        private static void VerifyCompilationMessagesBase(IEnumerable<LogInfo> messages, bool isEnabled)
```

Comment:
`VerifyCompilationMessagesBase.isEnabled` - meaning of this is not clear, looks like it should enable the verification itself.

Let's change it to `string expectedConcurrencyMessage` also to avoid the ternary calculation below.